0.Preprocess_Data/1.preprocessing.ipynb

```diff
@@ -6,16 +6,16 @@
    "id": "1ed18d57",
```

Comment:
<p>I think you could refactor this as so:</p><pre class="ql-syntax" spellcheck="false">columns_to_fill = ["Metadata_inducer1_concentration", "Metadata_inducer2_concentration", "Metadata_inhibitor_concentration"]
feature_df[columns_to_fill].fillna(0, inplace=True)
</pre><p> </p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/WayScience/pyroptosis_signature_data_analysis/pull/58/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='10'/>

Reply:
<p>implemented </p>